Packet++/src/GtpLayer.cpp

```diff
@@ -155,522 +155,521 @@ GtpV1Layer::GtpExtension GtpV1Layer::GtpExtension::createGtpExtension(uint8_t* d
 {
-    // get GTP header
-    gtpv1_header* header = getHeader();
-    if (header == NULL)
-    {
-        LOG_ERROR("Set sequence failed: GTP header is NULL");
-        return false;
-    }
+	// get GTP header
+	gtpv1_header *header = getHeader();
+	if (header == NULL)
+	{
+		LOG_ERROR("Set sequence failed: GTP header is NULL");
+		return false;
+	}
 
-     // if all flags are unset then create the GTP extra header
-    if (header->npduNumberFlag == 0 && header->sequenceNumberFlag == 0 && header->extensionHeaderFlag == 0)
-    {
-        if (!extendLayer(sizeof(gtpv1_header), sizeof(gtpv1_header_extra)))
-        {
-            LOG_ERROR("Set sequence failed: cannot extend layer");
-            return false;
-        }
-        header = getHeader();
-    }
+	// if all flags are unset then create the GTP extra header
+	if (header->npduNumberFlag == 0 && header->sequenceNumberFlag == 0 && header->extensionHeaderFlag == 0)
+	{
+		if(!extendLayer(sizeof(gtpv1_header), sizeof(gtpv1_header_extra)))
+		{
+			LOG_ERROR("Set sequence failed: cannot extend layer");
+			return false;
+		}
+		header = getHeader();
+	}
 
-    // get the extra header
-    gtpv1_header_extra* headerExtra = getHeaderExtra();
-    if (headerExtra == NULL)
-    {
-        LOG_ERROR("Set sequence failed: extra header is NULL");
-        return false;
-    }
+	// get the extra header
+	gtpv1_header_extra *headerExtra = getHeaderExtra();
+	if (headerExtra == NULL)
+	{
+		LOG_ERROR("Set sequence failed: extra header is NULL");
+		return false;
+	}
 
-    // set seq number
-    header->sequenceNumberFlag = 1;
-    headerExtra->sequenceNumber = htobe16(seqNumber);
+	// set seq number
+	header->sequenceNumberFlag = 1;
+	headerExtra->sequenceNumber = htobe16(seqNumber);
 
-    // extend GTP length
-    header->messageLength = htobe16(be16toh(header->messageLength) + sizeof(gtpv1_header_extra));
+	// extend GTP length
+	header->messageLength = htobe16(be16toh(header->messageLength) + sizeof(gtpv1_header_extra));
 
-    return true;
+	return true;
 }
 
-bool GtpV1Layer::getNpduNumber(uint8_t& npduNum)
+bool GtpV1Layer::getNpduNumber(uint8_t &npduNum) const
 {
-    gtpv1_header* header = getHeader();
-    gtpv1_header_extra* headerExtra = getHeaderExtra();
-    if (header != NULL && headerExtra != NULL && header->npduNumberFlag == 1)
-    {
-        npduNum = headerExtra->npduNumber;
-        return true;
-    }
+	gtpv1_header *header = getHeader();
+	gtpv1_header_extra *headerExtra = getHeaderExtra();
+	if (header != NULL && headerExtra != NULL && header->npduNumberFlag == 1)
+	{
+		npduNum = headerExtra->npduNumber;
+		return true;
+	}
 
-    return false;
+	return false;
 }
 
 bool GtpV1Layer::setNpduNumber(const uint8_t npduNum)
 {
-    // get GTP header
-    gtpv1_header* header = getHeader();
-    if (header == NULL)
-    {
-        LOG_ERROR("Set N-PDU failed: GTP header is NULL");
-        return false;
-    }
+	// get GTP header
+	gtpv1_header *header = getHeader();
```

Comment:
please move the `*` back to its original place. This is the convention in the rest of the project

Reply:
Done